alarm/CMakeLists.txt

```
@@ -0,0 +1,21 @@
+esma_set_this (OVERRIDE MAPL.alarm)
+
+set(srcs
+  SimpleAlarm.F90
+)
+
+list (APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
+
+if (BUILD_WITH_PFLOGGER)
+  find_package (PFLOGGER REQUIRED)
+endif ()
+
+esma_add_library(${this}
+  SRCS ${srcs}
+  DEPENDENCIES MAPL.shared PFLOGGER::pflogger ESMF::ESMF
```

```suggestion
  DEPENDENCIES MAPL.shared ESMF::ESMF
```
No dependency on pflogger is needed here.